src/plugins/data_source_management/public/components/popover_button/popover_button.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React from 'react';
+import { EuiButtonEmpty } from '@elastic/eui';
+import { i18n } from '@osd/i18n';
+
+interface PopoverButtonProps {
+  className: string;
+  label?: string;
+  onClick: () => void;
+}
+
+export const PopoverButton: React.FC<PopoverButtonProps> = ({ className, label, onClick }) => {
```

Comment:
Can we use a more specific name, i.e. DataSourceMenuButton

Reply:
Would do